test/ntp_client_test.rb

```diff
@@ -679,7 +683,7 @@
     end
 
     it "initializes restrict records" do
-      expect(subject.restrict_map.size).to eql(4)
+      expect(subject.restrict_map.size).to eql(3)
```

Comment:
Basically it only reads 3 restrict records because they are indexed by the "address" and 
```
restrict -4 default
restrict -6 default
```
are indexed by the "default" key. With the older lense there was an error which added "-4" as an address that is the reason of having 4 entries previously.